pandas/_libs/writers.pyx

```diff
@@ -36,21 +36,16 @@ def write_csv_rows(list data, ndarray data_index,
     cols : ndarray
     writer : object
     """
+    # In crude testing, N>100 yields little marginal improvement
```

Comment:
Hmm...that's not very encouraging to read.  Let's revert the initialization on `N` here until more substantive testing can be performed to prove this out.

Reply:
This comment isn’t new, just moved from a few lines down.

Reply:
Ah...gotcha.   I didn't realize 😂